website/src/components/DataTable/DataTable.tsx

```diff
@@ -124,7 +124,7 @@ export const DataTable = <T,>({
           <TableCaption pb={0}>{caption}</TableCaption>
           <Thead>
             {getHeaderGroups().map((headerGroup) => (
-              <Tr key={headerGroup.id} sx={{ "white-space": "normal" }}>
+              <Tr key={headerGroup.id} sx={{ whiteSpace: "normal" }}>
```

Comment:
is it possible to define the whitespace on the element directly? not `sx`?

Reply:
Not sure what you mean, sorry.  I was just fixing the obvious mistake here.

Reply:
I mean `<Tr key={headerGroup.id} whiteSpace="normal">`

Reply:
done